pygtfs/loader.py

```diff
@@ -82,9 +83,17 @@ def append_feed(schedule, feed_filename, strip_fields=True,
             if not record:
                 # Empty row.
                 continue
-
+            if gtfs_class == Agency:
+                # agency_id on routes.txt is only required if multiple agencies are defined in agency.txt
```

Comment:
If we do follow this route, can you add to the comment that agencies are always processed before routes? (That's true, but a casual reader might miss this)